kube_proxy/datadog_checks/kube_proxy/__init__.py

```diff
@@ -0,0 +1,10 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from . import kube_proxy
+
+KubeProxyCheck = kube_proxy.KubeProxyCheck
+
+__version__ = "1.0.0"
+
+__all__ = ['kube_proxy']
```

Comment:
this should be `__all__ = ['KubeProxyCheck']` to allow `from datadog_checks.kube_proxy import KubeProxyCheck`